eZ/Publish/Core/Persistence/Legacy/Content/FieldValue/Converter/UserConverter.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\Core\Persistence\Legacy\Content\FieldValue\Converter;
+
+use eZ\Publish\Core\Persistence\Legacy\Content\FieldValue\Converter;
+use eZ\Publish\Core\Persistence\Legacy\Content\StorageFieldDefinition;
+use eZ\Publish\Core\Persistence\Legacy\Content\StorageFieldValue;
+use eZ\Publish\SPI\Persistence\Content\FieldValue;
+use eZ\Publish\SPI\Persistence\Content\Type\FieldDefinition;
+
+class UserConverter implements Converter
```

Comment:
In general it worries me a bit. This converter was not implemented on purpose. Entire processing is done by UserService and its dedicated persistence handlers and gateways. Are you sure this doesn't break anything?